pyfrc/mains/cli_test.py

```diff
@@ -37,6 +38,13 @@ def __init__(self, parser=None):
                 action="store_true",
                 help="Use pyfrc's builtin tests if no tests are specified",
             )
+            parser.add_argument(
+                "-m",
+                "--multiprocess",
```

Comment:
This option name isn't great, open to suggestions?

Reply:
multiprocess makes it sound like the tests are run in parallel.
Something like `isolated` would be better.
Then, if you switch to it as the default and want an optout, it can be `notisolated`

Reply:
`isolated` is probably better than the other alternative `out-of-process` I was thinking of